config.py

```diff
@@ -23,12 +25,17 @@ class FilterRule(BaseModel):
 class Settings(BaseSettings):
     port_client_id: str
     port_client_secret: str
-    slack_webhook_url: str
+    slack_webhook_url: str = ""
+    jira_project_id: str = ""
+    jira_api_endpoint: str = "https://jira.com"
+    jira_email: str = ""
+    jira_resolve_transition_id: str = ""
+    jira_token: str = ""
     port_region: str = "eu"
     blueprint: str
     scorecard: str
     filter_rule: Union[FilterRule, str, None] = Field(default=None)
-    message_kind: MessageKind = MessageKind.scorecard_reminder
+    operation_kind: OperationKind = OperationKind.scorecard_reminder
     target_kind: TargetKind = TargetKind.slack
```

Comment:
lets split it to classes each for each type of operator, it will be cleaner to understand the related configuration for each.

Reply:
Let's discuess it